requirements_test_all.txt

```diff
@@ -1729,6 +1729,9 @@ python-ecobee-api==0.2.17
 # homeassistant.components.fully_kiosk
 python-fullykiosk==0.0.12
 
+# homeassistant.components.sms
+# python-gammu==3.2.4
```

Comment:
Why this is commented out?
If not needed then should we remove it?

Reply:
It's commented out because `python-gammu` relies on additional Linux packages. The comment is removed when building wheels and is also removed in `.github/workflows/ci.yaml` when running tests.